pkg/agent/prommetrics.go

```diff
@@ -217,6 +220,13 @@ func makeGlobalMetrics() (GlobalMetrics, *prometheus.Registry) {
 				Help: "Number of times (*core.State).NextActions() has been called",
 			},
 		)),
+
+		scalingLatency: *util.RegisterMetric(reg, prometheus.NewHistogramVec(
+			prometheus.HistogramOpts{
+				Name: "autoscaling_agent_scaling_latency_seconds",
+				Help: "End-to-end scaling latency",
```

Comment:
We need to also set the buckets here, right?

Reply:
Yeah, the defaults have low quite resolution.